### PR TITLE
Fix docker image push CI tests

### PR DESCRIPTION
**Pull request by ivanezeigbo**

### CODE REVIEW

File: .github/workflows/docker_image_push.yml [hunk 1]
```diff
@@ -44,10 +44,14 @@ jobs:
...
+      uses: docker/login-action@v1 
+      with:
+        username: ${{secrets.DOCKER_USER}}
+        password: ${{secrets.DOCKER_PASS}}
+    - name: Build and Push to Docker
+      uses: docker/build-push-action@v2
+      with:
+        context: .
+        push: true
+        tags: bittensor/bittensor:subtensor_tests
```
Comment (unconst) [inline .github/workflows/docker_image_push.yml]: why do we have this tag here? bittensor:subtensor_tests? shouldn't it just be bittensor:bittensor.

Comment (ivanezeigbo) [reply .github/workflows/docker_image_push.yml]: @unconst do you mean `bittensor/bittensor:latest` ? So following the circle ci config, the push image script seems to target `bittensor/bittensor:subtensor_tests` 

`./scripts/build_protos.sh`
`docker build -t bittensor/bittensor:subtensor_tests -f Dockerfile .`
`docker push bittensor/bittensor:subtensor_tests`

Should I use `bittensor/bittensor:latest` instead?

Comment (unconst) [reply .github/workflows/docker_image_push.yml]: Yes I think you should yes. @shibshib  can you comment on this? Or was Parralax the prev owner here

Comment (shibshib) [reply .github/workflows/docker_image_push.yml]: Definitely some remnant of parallaxs code, should use bittensor/bittensor:latest

Comment (shibshib) [reply .github/workflows/docker_image_push.yml]: I deleted that subtensor thing

Comment (ivanezeigbo) [reply .github/workflows/docker_image_push.yml]: made the necessary changes. @unconst @shibshib please review

**Review by shibshib — APPROVED** [review]

**Pull request merged**